types.go

```diff
@@ -250,10 +252,18 @@ func isWaitGroup(t types.Type) bool {
 }
 
 func isProphId(t types.Type) bool {
+	// "dereference" alias types to get down to *prophId
```

Comment:
What about [`t = types.Unlias(t)`](https://pkg.go.dev/go/types#Unalias) instead of this loop?

Reply:
Good find! That's exactly the right API.